backend/src/main/java/com/karankumar/bookproject/backend/service/ImportServiceDelegate.java

```diff
@@ -0,0 +1,25 @@
+package com.karankumar.bookproject.backend.service;
+
+import static java.util.Collections.emptyList;
+
+import com.karankumar.bookproject.backend.model.bookImport.libraryThing.LibraryThingBookDataImport;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class ImportServiceDelegate {
+
+  private final LibraryThingBookImportService libraryThingBookImportService;
```

Comment:
Do we want to autowire these in?

Reply:
From Spring Boot 1.4+ if the class has one constructor and injection can be determined, you don't need to use autowired annotation at all 

https://docs.spring.io/spring-boot/docs/current/reference/html/using.html#using.spring-beans-and-dependency-injection

Reply:
Thanks